### PR TITLE
Update permissions.md

### PR DESCRIPTION
**Pull request by theredcat**
Use values from https://api.cloudflare.com/client/v4/user/tokens/permission_groups

```
{
"id":"c4a30cd58c5d42619c86a3c36c441e2d",
"name":"Logs Read",
"description":"Grants read access to logs and logpush jobs",
"scopes":[
"com.cloudflare.api.account.zone"
]
},
{
"id":"6a315a56f18441e59ed03352369ae956",
"name":"Logs Read",
"description":"Grants read access to logs and logpush jobs",
"scopes":[
"com.cloudflare.api.account"
]
},
{
"id":"3e0b5820118e47f3922f7c989e673882",
"name":"Logs Write",
"description":"Grants write access to logpush jobs",
"scopes":[
"com.cloudflare.api.account.zone"
]
},
{
"id":"96163bd1b0784f62b3e44ed8c2ab1eb6",
"name":"Logs Write",
"description":"Grants write access to logpush jobs",
"scopes":[
"com.cloudflare.api.account"
]
},
```

### CODE REVIEW

**Comment by theredcat** [comment]
@renandincer

**Review by deuill — APPROVED** [review]

**Comment by theredcat** [comment]
@deuill @KimJ15 @renandincer Bump !

File: products/api/src/content/tokens/create/permissions.md [hunk 1]
```diff
@@ -57,8 +57,8 @@ Below is a list of all the Token Permissions that are available to use. The curr
 |Load Balancers Write|6d7f2f5f5b1d4a0e9081fdc98d432fd1|Grants write access to load balancers and associated resources|com.cloudflare.api.account.zone|
 |Load Balancing: Monitors and Pools Read|9d24387c6e8544e2bc4024a03991339f|Grants read access to account level load balancer resources|com.cloudflare.api.account|
 |Load Balancing: Monitors and Pools Write|d2a1802cc9a34e30852f8b33869b2f3c|Grants write access to account level load balancer resources|com.cloudflare.api.account|
-|Logs Read|c4a30cd58c5d42619c86a3c36c441e2d|Grants read access to logs and logpush jobs|com.cloudflare.api.account.zone|
```
Comment (deuill) [inline products/api/src/content/tokens/create/permissions.md:60]: I realized I approved earlier but there's actually *two* sets of permission groups with the same name (and different IDs): one for `com.cloudflare.api.account.zone` scopes, and one for `com.cloudflare.api.account` scopes. That is to say, we shouldn't be removing any rows here, but simply adding the new ones.

Comment (theredcat) [reply products/api/src/content/tokens/create/permissions.md:60]: That's actually my point :
  - one of the permission set is zone-level
  - one is account-level

Zone level is fine in the doc.

But for account-level, it's the zone-level permission group ID that is used in the doc, which is wrong.

Here is a working example. Notice the 2 different Logs Write permissions ID used :

```json
{
  "name": "session",
  "not_before": "xxxxxxxxxxxxxxx",
  "expires_on": "xxxxxxxxxxxxxxxxxxx",
  "policies": [
    {
      "effect": "allow",
      "resources": {
        "com.cloudflare.api.account.zone.*": "*"
      },
      "permission_groups": [
        {
          "id": "c8fed203ed3043cba015a93ad1616f1f",
          "name": "Zone Read"
        },
        {
          "id": "82e64a83756745bbbb1c9c2701bf816b",
          "name": "DNS Read"
        },
        {
          "id": "7b7216b327b04b8fbc8f524e1f9b7531",
          "name": "SSL and Certificates Read"
        },
        {
          "id": "3e0b5820118e47f3922f7c989e673882",
          "name": "Logs Write"
        },
        {
          "id": "517b21aee92c4d89936c976ba6e4be55",
          "name": "Zone Settings Read"
        },
        {
          "id": "4ec32dfcb35641c5bb32d5ef1ab963b4",
          "name": "Firewall Services Read"
        }
      ]
    },
    {
      "effect": "allow",
      "resources": {
        "com.cloudflare.api.account.xxxxxxxxxxxxxxxxxxxxxxxxx": "*"
      },
      "permission_groups": [
        {
          "id": "7cf72faf220841aabcfdfab81c43c4f6",
          "name": "Billing Read"
        },
        {
          "id": "96163bd1b0784f62b3e44ed8c2ab1eb6",
          "name": "Logs Write"
        },
        {
          "id": "de7a688cc47d43bd9ea700b467a09c96",
          "name": "Account Firewall Access Rules Read"
        }
      ]
    }
  ]
}
```

Comment (deuill) [reply products/api/src/content/tokens/create/permissions.md:60]: Ah, apologies, I completely missed the fact that the permission group was re-defined elsewhere. Thanks for following up!

**Review by deuill — APPROVED** [review]

**Comment by garrettgalow** [comment]
@deadlypants1973 can you merge? This has been reviewed by the appropriate folks on the technical side.

**Review by garrettgalow — APPROVED** [review]

**Review by KimJ15 — APPROVED** [review]

**Pull request merged**